### PR TITLE
refine the standardize-error-codes-and-messages RFC 

### PR DESCRIPTION
**Pull request by lucklove**
### What problem does this PR solve?

refine the standardize-error-codes-and-messages RFC 
Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### CODE REVIEW

File: docs/design/2020-05-08-standardize-error-codes-and-messages.md [hunk 1]
```diff
@@ -163,8 +163,8 @@ In addition, the error codes should be append only in case of conflict between v
 
 In the discussion above, an error has at least 4 parts:
 - The error code: it's the identity of an error
-- The error field: it's the error itself the user can view in TiDB system
-- the message field: the description of the error, what happened and why happend?
+- The message field: it's the error itself the user can view in TiDB system
+- the description field: the description of the error, what happened and why happend?
```
Comment (overvenus) [inline docs/design/2020-05-08-standardize-error-codes-and-messages.md:167]:
```suggestion
- the description field: the description of the error, what happened and why happened?
```

**Comment by overvenus** [comment]
Could you explain what's the difference between message and description?

**Comment by lonng** [comment]
> Could you explain what's the difference between message and description?

The `message` is an example message corresponding to the error code. The `description` is the expanded detail of why this error occurred. And the `workaround` is used to teach the users how to solve the error if occurring in the real environment.

**Comment by lucklove** [comment]
Closing this since https://github.com/pingcap/tidb/pull/17446 has a more detailed update.

**Pull request closed without merging**